src/libPMacc/include/particles/ParticlesBase.kernel

```diff
@@ -389,11 +377,13 @@ struct KernelFillGaps
 
             if ( counterGaps == 0 )
             {
-                updateCellsForOneFrame<TileSize>(cellEntryPoint_sh, cellCount_sh, firstFrame, frameNr);
+                if (updateCellList)
+                    kernel::UpdateCellListOfFrame{}.template operator()<TileSize>(*cellEntryPoint_sh, *cellCount_sh, firstFrame, *frameNr);
```

Comment:
Here is an other example I documented years ago: https://github.com/ax3l/cpp_snippets/blob/master/template_error/template.cpp